src/countOccurrences.js

```diff
@@ -18,7 +18,13 @@
  *
  */
 function countOccurrences(phrase, letter) {
-  // write code here
+  let count = 0;
+  for (let i = 0; i < phrase.length; i++) {
+    if (phrase[i].toUpperCase() === letter.toUpperCase()) {
+      count++;
+    }
+  }
+  return count;
```

Comment:
new line before return
```suggestion

  return count;
```